src/unitful.jl

```diff
@@ -68,7 +68,13 @@ unitful(u::Vector{UnitPart}) = prod(unitful.(u))
 Computes a properly unitful value from a value-unit pair stored in the model
 `m`.
 """
-unitful(m::Model, val::Tuple{Float64,String}) = unitful(m.units[val[2]]) * val[1]
+function unitful(m::Model, val::Tuple{Float64,String})
+    if haskey(m.units, val[2])
+        unitful(m.units[val[2]]) * val[1]
+    else 
+        UNITFUL_KIND_STRING[val[2]] * val[1]
```

Comment:
Is there anything in the specification about this?

Reply:
On the top of page 44 of the l3v2 spec, it says that the symbols you have in that dict are reserved, but I haven't read particularly closely

Reply:
the defaulting here probably needs to be a bit more complex. It should be able to reasonably default to e.g. substance_units for substance-ish stuff. That needs a slight modification of the API because the `unitful` function can't see where the Tuple comes from (if it is a volume or flux or what).

I'd suggest going with something like
```julia
unitful(m,  (0.1,""), :volume)
```
that defaults to 0.1 of the default volume units.

Reply:
Anyway, I didn't find any notice of this kind of defaulting in the SBML docs (the closest "reserved" word relates to base units). The spec is quite unhelpful on specifying on how to interpret the data.

Can you please point out the precise sections&paragraphs that warrant that this defaulting is right?

Reply:
whoa, is the model online? (does it pass the SBML validator? :D )